### PR TITLE
Fix small pine leafdecay: New schematics, new leafdecay radius 

### LINKED ISSUE #2277: Pine tree leaves do not decay properly
The pointy top of 2 blocks of the pine tree does not decay and that is because the decay radius is set to 2 (mapgen v7) while the pointy top is of 4 blocks length.

Two possible solutions:
Change the radius to 4 **but** this will cause a problem in case of two adjacent trees.
Shorten the pointy top to 2 blocks length so it decays with the rest of the tree.

https://github.com/minetest/minetest_game/blob/da10af919eddc8f7e73c14f802290a5ea8f7b5f0/mods/default/nodes.lua#L2775-L2815

https://github.com/minetest/minetest_game/blob/da10af919eddc8f7e73c14f802290a5ea8f7b5f0/mods/default/trees.lua#L263-L270

screenshot_20181231_024104
screenshot_20181231_024347

### PR DESCRIPTION
**Pull request by paramat**
Previously the top 2 pine needles nodes of small pines were not
decaying.

Extend trunk upwards by 1 node and increase leafdecay radius to 3.
////////////////////

screenshot_20190103_030529

^ New small pine cross section

Closes #2277

### CODE REVIEW

File: mods/default/nodes.lua [hunk 1]
```diff
@@ -2847,43 +2847,31 @@ minetest.register_node("default:cloud", {
...
-		trunks = {"default:pine_tree"},
-		leaves = {"default:pine_needles"},
-		radius = 2,
-	})
 end
 
+default.register_leafdecay({
+	trunks = {"default:tree"},
+	leaves = {"default:apple", "default:leaves"},
+	radius = 2,
```
[the remark this answers is not in the document]
Comment (paramat) [reply mods/default/nodes.lua]: You mean old non-mgv6 trees.
Correct it does, silly me, will fix.

**Comment by paramat** [comment]
Merging as fairly trivial and thumbed-up in https://github.com/minetest/minetest_game/issues/2277#issuecomment-450704063

**Comment by Ezhh** [comment]
@paramat If the schematic has changed, don't forget to update schematic_tables.txt.

**Comment by paramat** [comment]
Aaargh sorry, will do.

**Comment by paramat** [comment]
Done #2285

**Pull request merged**